Not sure if you're aware, but we can test this before merging:
```
matwolf in bbpv2 in ~ via  v14.13.0 via 🐍 v2.7.5
❯ unset MODULEPATH

matwolf in bbpv2 in ~ via  v14.13.0 via 🐍 v2.7.5
❯ . /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/1246/config/modules.sh

matwolf in bbpv2 in ~ via  v14.13.0 via 🐍 v2.7.5
❯ module load unstable py-atlalign
Autoloading python/3.8.3
Autoloading hpe-mpi/2.22.hmpt
Autoloading ffmpeg/4.2.2
Autoloading cudnn/8.0.3.33-11.0-linux-x64
Autoloading cuda/11.0.2

matwolf in bbpv2 in ~ via  v14.13.0 via 🐍 v3.8.3
❯ python
Python 3.8.3 (default, Jan  6 2021, 17:13:49)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
Could not open PYTHONSTARTUP
FileNotFoundError: [Errno 2] No such file or directory: '/gpfs/bbp.cscs.ch/home/matwolf/.pystartup'
>>> from atlalign.ml_utils import load_model, merge_global_local
2021-08-09 14:22:11.913890: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-statsmodels-0.10.2-ifyiby/lib/python3.8/site-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
  import pandas.util.testing as tm
/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-patsy-0.5.1-euazqb/lib/python3.8/site-packages/patsy/constraint.py:13: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
  from collections import Mapping
/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-antspyx-0.2.7-gtysm6/lib/python3.8/site-packages/ants/core/ants_image.py:593: DeprecationWarning:

inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()
```
repeats until:
```
Traceback (most recent call last):
  File "/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-atlalign-0.6.0-tf5yum/lib/python3.8/site-packages/atlalign/metrics.py", line 42, in <module>
    import lpips_tf
ModuleNotFoundError: No module named 'lpips_tf'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-atlalign-0.6.0-tf5yum/lib/python3.8/site-packages/atlalign/ml_utils/__init__.py", line 23, in <module>
    from atlalign.ml_utils.callbacks import MLFlowCallback, get_mlflow_artifact_path  # noqa
  File "/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-atlalign-0.6.0-tf5yum/lib/python3.8/site-packages/atlalign/ml_utils/callbacks.py", line 30, in <module>
    from atlalign.metrics import evaluate_single
  File "/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-atlalign-0.6.0-tf5yum/lib/python3.8/site-packages/atlalign/metrics.py", line 44, in <module>
    raise ModuleNotFoundError(
ModuleNotFoundError:
        LPIPS-TensorFlow required but not found.

        Please install it by running the following command:
        $ pip install git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf
```
```

--- a/var/spack/repos/builtin/packages/py-atlalign/lpips.patch
+++ b/var/spack/repos/builtin/packages/py-atlalign/lpips.patch
@@ -1,0 +1,65 @@
+commit d86b99e60c187e00732af082b3f79b6d46bbf585
+Author: Matthias Wolf <matthias.wolf@epfl.ch>
+Date:   Mon Aug 9 14:36:58 2021 +0200
+
+    dependencies: move lpips import into function
+    
+    This makes the lpips functionality somewhat optional by moving the
+    import from the module scope into the function where it is used.  Should
+    simplify installation.
+
+diff --git a/README.md b/README.md
+index e87f781..e7a29a0 100644
+--- a/README.md
++++ b/README.md
+@@ -80,7 +80,8 @@ All details related to installation and logic are described in the
+ 
+ Some of the functionalities of `atlalign` depend on the [TensorFlow implementation
+ of the Learned Perceptual Image Patch Similarity (LPIPS)](https://github.com/alexlee-gk/lpips-tensorflow). Unfortunately, the
+-package is not available on PyPI and must be installed manually as follows.
++package is not available on PyPI and must be installed manually as follows
++for full functionality.
+ ```shell script
+ pip install git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf
+ ```
+diff --git a/atlalign/metrics.py b/atlalign/metrics.py
+index 90563ac..4a0a6ed 100644
+--- a/atlalign/metrics.py
++++ b/atlalign/metrics.py
+@@ -38,18 +38,6 @@ from functools import wraps
+ 
+ import ants
+ 
+-try:
+-    import lpips_tf
+-except ModuleNotFoundError as err:
+-    raise ModuleNotFoundError(
+-        """
+-        LPIPS-TensorFlow required but not found.
+-
+-        Please install it by running the following command:
+-        $ pip install git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf
+-        """
+-    ) from err
+-
+ import numpy as np
+ import pandas as pd
+ import tensorflow as tf
+@@ -996,6 +984,17 @@ def perceptual_loss_img(y_true, y_pred, model="net-lin", net="vgg"):
+     We use the decorator just to make sure we do not run out of memory during a forward pass. Also,
+     its fully convolutional but if the images are too small then might run into issues.
+     """
++    try:
++        import lpips_tf
++    except ModuleNotFoundError as err:
++        raise ModuleNotFoundError(
++            """
++            LPIPS-TensorFlow required but not found.
++
++            Please install it by running the following command:
++            $ pip install git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf
++            """
++        ) from err
+     # gray2rgb
+     y_true = np.stack((y_true,) * 3, axis=-1)
+     y_pred = np.stack((y_pred,) * 3, axis=-1)

--- a/var/spack/repos/builtin/packages/py-atlalign/package.py
+++ b/var/spack/repos/builtin/packages/py-atlalign/package.py
@@ -27,5 +27,6 @@ class PyAtlalign(PythonPackage):
     depends_on('py-scikit-image@0.16.0:', type=('run'))
     depends_on('py-scikit-learn@0.20.2:', type=('run'))
     depends_on('py-scipy', type=('run'))
-    depends_on('py-tensorflow@2.4.0:', type=('run'))
-    depends_on('py-tensorflow-addons', type=('run'))
+    # Addons need to be in lockstep with TF
+    depends_on('py-tensorflow@2.4', type=('run'))
+    depends_on('py-tensorflow-addons@0.12', type=('run'))

--- a/var/spack/repos/builtin/packages/py-atlalign/package.py
+++ b/var/spack/repos/builtin/packages/py-atlalign/package.py
@@ -30,3 +30,5 @@ class PyAtlalign(PythonPackage):
     # Addons need to be in lockstep with TF
     depends_on('py-tensorflow@2.4', type=('run'))
     depends_on('py-tensorflow-addons@0.12', type=('run'))
+
+    patch('lpips.patch', when='@0.6.0')

--- a/var/spack/repos/builtin/packages/py-tensorflow-addons/package.py
+++ b/var/spack/repos/builtin/packages/py-tensorflow-addons/package.py
@@ -34,7 +34,7 @@ class PyTensorflowAddons(Package):
     depends_on('py-pip', type='build')
 
     depends_on('py-setuptools', type='build')
-    depends_on('py-tensorflow@2.3:2.5', type=('run'), when='@0.13:')
+    depends_on('py-tensorflow@2.5', type=('run'), when='@0.13:')
     depends_on('py-tensorflow@2.4.0:2.4.2', type=('run'), when='@0.12.1')
     depends_on('py-typeguard@2.7:', type=('run'))
     # no versions for Mac OS added

